x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/outlier_exploration/outlier_exploration.tsx

```diff
@@ -55,57 +69,182 @@ export const OutlierExploration: FC<ExplorationProps> = React.memo(({ jobId }) =
     jobConfig !== undefined ? getFeatureCount(jobConfig.dest.results_field, tableItems) : 1
   );
 
+  const [expandedRowItem, setExpandedRowItem] = useState<DataFrameAnalyticsListRow | undefined>(
+    undefined
+  );
+
+  const fetchStats = async () => {
+    const analyticsConfigs = await ml.dataFrameAnalytics.getDataFrameAnalytics(jobId);
+    const analyticsStats = await ml.dataFrameAnalytics.getDataFrameAnalyticsStats(jobId);
+
+    const config = analyticsConfigs.data_frame_analytics[0];
+    const stats = isGetDataFrameAnalyticsStatsResponseOk(analyticsStats)
+      ? analyticsStats.data_frame_analytics[0]
+      : undefined;
+
+    if (stats === undefined) {
+      return;
+    }
+
+    const newExpandedRowItem: DataFrameAnalyticsListRow = {
+      checkpointing: {},
+      config,
+      id: config.id,
+      job_type: Object.keys(config.analysis)[0] as DataFrameAnalysisConfigType,
+      mode: DATA_FRAME_MODE.BATCH,
+      state: stats.state,
+      stats,
+    };
+
+    setExpandedRowItem(newExpandedRowItem);
+  };
+
+  useEffect(() => {
+    fetchStats();
+  }, [jobConfig?.id]);
+
   return (
-    <EuiPanel data-test-subj="mlDFAnalyticsOutlierExplorationTablePanel">
-      {jobConfig !== undefined && needsDestIndexPattern && (
-        <IndexPatternPrompt destIndex={jobConfig.dest.index} />
-      )}
-      <EuiFlexGroup
-        alignItems="center"
-        justifyContent="spaceBetween"
-        responsive={false}
-        gutterSize="s"
-      >
-        <EuiFlexItem grow={false}>
-          <ExplorationTitle title={explorationTitle} />
-        </EuiFlexItem>
-        {jobStatus !== undefined && (
-          <EuiFlexItem grow={false}>
-            <span>{getTaskStateBadge(jobStatus)}</span>
-          </EuiFlexItem>
-        )}
-      </EuiFlexGroup>
-      <EuiHorizontalRule margin="xs" />
+    <>
       {(columnsWithCharts.length > 0 || searchQuery !== defaultSearchQuery) &&
         indexPattern !== undefined && (
           <>
-            <EuiFlexGroup justifyContent="spaceBetween">
-              <EuiFlexItem>
-                <ExplorationQueryBar indexPattern={indexPattern} setSearchQuery={setSearchQuery} />
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiSpacer size="s" />
-                <ColorRangeLegend
-                  colorRange={colorRange}
-                  title={i18n.translate(
-                    'xpack.ml.dataframe.analytics.exploration.colorRangeLegendTitle',
-                    {
-                      defaultMessage: 'Feature influence score',
-                    }
+            <ExplorationQueryBar indexPattern={indexPattern} setSearchQuery={setSearchQuery} />
```

Comment:
Did you consider moving the query bar below the Analysis section? I can't decide where it fits best - on the one hand most views place the query bar at the top, but on the other hand the query bar would sit well directly above the panels on which it has an effect (in this view just the result table for now).

Reply:
I thought about it during design phase and had a look at other plugin/kibana pages, most have them at the very top, even if some sections below it might not be in complete context of the search bar. There are also other plugins that keep the search bar pinned at the top when you scroll down the page which is an idea I'd like to explore but seemed overkill for this iteration. With these considerations I went for having it at the top for this PR.